docs/tests-system.md

```diff
@@ -0,0 +1,100 @@
+---
+category: CoreDevelop
+---
+# Writing system tests
```

Comment:
We do not want to encourage developers to write system tests and it is quite complex as well. Therefore under core development